app/code/Magento/InventoryCatalog/Test/_files/source_items_on_mixed_sources.php

```diff
@@ -0,0 +1,62 @@
+<?php
```

Comment:
@phoenix128  why do you need to create own fixture `source_items_on_mixed_sources.php`
if we already have existing `source_items.php` ?